src/genie/libs/parser/iosxe/tests/test_show_ap.py

```diff
@@ -0,0 +1,239 @@
+import unittest
+from unittest.mock import Mock
+
+from genie.metaparser.util.exceptions import SchemaEmptyParserError
+from genie.libs.parser.iosxe.show_ap import ShowApSummary
+
+
+# ===============================
+# Unit test for 'show ap summary'
+# ===============================
+class TestShowApSummary(unittest.TestCase):
+    """Unit test for 'show ap summary'"""
+
+    maxDiff = None
+    empty_output = {'execute.return_value': ''}
+    golden_parsed_output1 = {
+        "ap_summary": {
+            "ap_neighbor_count": 149,
+            "a121-cap22": {
+                "slots_count": 2,
+                "ap_model": "9130AXI",
+                "ethernet_mac": "a4b2.3291.9b28",
+                "radio_mac": "2c57.4119.a060",
+                "location": "Fab A  UK",
```

Comment:
I believe `UK` needs to be separated like below:
```
"location": "Fab A",
"country": "UK",
```

Reply:
@tahigash, I've seen different examples for this. It's possible that the input is a free form text that may not include country. 
`Fab A  UK`
`UK HUB/Fab A - S  UK`
`UK HUB/Fab B - S  UK `
`default location`

Reply:
ok. thanks for the info. if so, let's keep as is.